<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgParamString</class>
 <widget class="QWidget" name="DlgParamString">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>47</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="bty">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>172</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>142</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="rb_Y">
      <property name="text">
       <string>是</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="rb_N">
      <property name="text">
       <string>否</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>172</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
